requirements.txt

```diff
@@ -21,3 +21,7 @@ django-reversion-compare==0.5.5
 diff-match-patch==20121119
 django-enumfield==1.2.1
 django-bootstrap3==6.1.0
+pyldap==2.4.25.1
+gunicorn
```

Comment:
We could also install gunicorn in the TeamCity deploy script, just like New Relic?